config/locales/en.yml

```diff
@@ -22,7 +22,7 @@ en:
         mobile_field: 'If you wish to have your secure one-time password sent to your mobile phone, enter the number here.'
         contact_info: Contact information
         already_have_account_html: Already have an account?
-        need_password: We need your current password to confirm your changes.
```

Comment:
I believe we should a bit less casual here (i.e., not using "We" unless we really need to)